notebooks/EDA Climate.ipynb

```diff
@@ -0,0 +1,1821 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    decomposition = seasonal_decompose(df_ocean, model='additive', period=4)
</pre><p> </p><p><code>seasonal_decompose</code> does "classical decomposition" via moving averages. <a href="https://otexts.com/fpp3/classical-decomposition.html" target="_blank">See here</a>., It's better to use a statistical method like STL or ARIMA-X13-SEATS. I like STL. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jessegrabowski/laos-climate-change/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>